app/components/generate-report-button.tsx

```diff
@@ -87,45 +92,58 @@ export default function GenerateReportButton({
             <>
               <ModalHeader className="flex flex-col gap-1">Generate report</ModalHeader>
               <ModalBody>
-                <Autocomplete
-                  allowsCustomValue
-                  errorMessage={resultProjectsError?.message}
-                  inputValue={projectName}
-                  isDisabled={isPending}
-                  isLoading={isResultProjectsLoading}
-                  items={Array.from(new Set([...projects, ...(resultProjects ?? [])])).map((project) => ({
-                    label: project,
-                    value: project,
-                  }))}
-                  label="Project name"
-                  labelPlacement="outside"
-                  placeholder="leave empty if not required"
-                  variant="bordered"
-                  onInputChange={(value) => setProjectName(value)}
-                  onSelectionChange={(value) => value && setProjectName(value?.toString() ?? '')}
-                >
-                  {(item) => <AutocompleteItem key={item.value}>{item.label}</AutocompleteItem>}
-                </Autocomplete>
-                <Input
-                  fullWidth
-                  isClearable
-                  label="Custom report name"
-                  labelPlacement="outside"
-                  maxLength={36}
-                  placeholder="leave empty if not required"
-                  value={customName}
-                  variant="bordered"
-                  onChange={(e: { target: { value: any } }) => setCustomName(e.target.value ?? '')}
-                  onClear={() => setCustomName('')}
-                />
+                {generationError ? (
+                  <div className="flex flex-col gap-2">
+                    <p className="text-danger font-semibold">Report generation failed:</p>
+                    <pre className="bg-danger-50 border border-danger-200 rounded-lg p-4 text-sm text-danger-700 font-mono whitespace-pre-wrap break-words overflow-auto max-h-96 select-text">
+                      {generationError}
+                    </pre>
+                  </div>
```

Comment:
Alert component could be used to display error state: https://www.heroui.com/docs/components/alert